pkg/sink/sink_test.go

```diff
@@ -466,6 +491,11 @@ func Test_HandleEvent(t *testing.T) {
 		if err = json.Unmarshal(body, &gotPipelineResource); err != nil {
 			t.Errorf("Error unmarshalling body as pipelineResource: %s \n%s", string(body), err)
 		}
+		if gotPipelineResource.ObjectMeta.Labels["tekton.dev/triggers-eventid"] == "" {
```

Comment:
At this point, the API server takes over and there is no way for us to validate this.

Reply:
That's true.